ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/JsonProperties.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.serde.json;
+
+import com.google.common.collect.ImmutableSet;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import io.confluent.ksql.serde.connect.ConnectProperties;
+import java.util.Map;
+
+public class JsonProperties extends ConnectProperties {
+
+  static final ImmutableSet<String> SUPPORTED_PROPERTIES = ImmutableSet.of();
```

Comment:
Why do we need to introduce this class if the set of supported properties is empty?

Reply:
1. Json extended `ConnectFormat`, so I thought it might make sense to create `JsonProperties` extends `ConnectProperties`?
2. Both Json and JsonSchema format call https://github.com/confluentinc/ksql/blob/master/ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonSerdeFactory.java#L40 to create serializer atm. I would still like to maintain the behavior and pass a property object of Json or JsonSchema to that class later. That's why I create `JsonProperties` and let `JsonSchemaProperties` extend it so that I can pass both `JsonProperties` and `JsonSchemaProperties` to `KsqlJsonSerdeFactory`

That being said, I guess 2 is a stronger reason but there should be other ways to solve it as well.